test/steps/image_registry.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package steps
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/cucumber/godog"
+	"github.com/kiegroup/kogito-cloud-operator/test/config"
+	"github.com/kiegroup/kogito-cloud-operator/test/framework"
+)
+
+// registerImageRegistrySteps register all existing image registry steps
+func registerImageRegistrySteps(s *godog.Suite, data *Data) {
+	s.Step(`^Local example service "([^"]*)" is deployed to image registry, image tag stored as variable "([^"]*)"`, data.localServiceDeployedToImageRegistry)
+}
+
+// Build local service
+func (data *Data) localServiceDeployedToImageRegistry(contextDir, imageTagContextKey string) error {
+	projectLocation := data.KogitoExamplesLocation + "/" + contextDir
+	// projectLocation := "/home/ksuta/src/github.com/kiegroup/submarine-examples/process-quarkus-example"
```

Comment:
My debugging leftover :) can be removed.

Reply:
Didn't see it!. Removed.